gems/pending/spec/miq_vm/miq_azure_vm_image_spec.rb

```diff
@@ -0,0 +1,81 @@
+require_relative './miq_vm_instance_methods_shared_examples'
+require 'ostruct'
+require 'azure-armrest'
+require 'MiqVm/miq_azure_vm'
+require 'vcr'
+
+describe MiqAzureVm do
+  before(:all) do
```

Comment:
What's wrong with `before(:all)`? It seems like a reasonable way to do test instance specific initialization. The TestEnvHelper implements test instance specific configuration - where cassettes are recorded, parameter settings, what gets filtered from the cassettes and how.

You should look at: https://github.com/ManageIQ/manageiq/pull/7341

This is an extension of the `secrets` concept used in the vmdb specs. I think those specs would benefit from this as well.